### ISSUE TITLE
use @Deprecated

### ISSUE THREAD

**Issue by StanLepunK** [processing/processing#5216]
In the future of Processing there is project to print a message if a method, class has marked by deprecated ?

```
void setup() {
  test() ;
}
@Deprecated
void test() {
}
```

**Comment by benfry** [comment]
I don't quite understand the question—can you rephrase?

**Comment by StanLepunK** [comment]
Sorry for my english ! my question, it's if there is a plan for the future to can used annotation like `@Deprecated` in Processing, like we can used `@Override`for example. I ask that because I used Processing to code often and sometime I change the name of method, but I don't want remove it, because this one can be used in other part of program, So if I understand well the function of `@Deprecated`you can received a message if this one is used. So if if `@override` is active in Processing, maybe it's possible for `@Deprecated`and few other annotation. I hope is clearest ?

**Comment by macbrayne** [comment]
Hi there,

Processing (v3.5.3 on Windows 10) does not warn when using `@Deprecated` methods however a program can’t run if a `@Override` annotation doesn’t actually override anything.
Behavior when using `@Override` and nothing is being overridden: The method […] must override or implement a supertype method
Behavior when using a deprecated method: No warning is given when trying to run the program.

Example code:

```java
void setup() {
  
}

void draw() {
  imold();
}

@Deprecated
void imold() {
  // Does old things
}
```
The expected behavior is warning the user via the console if he uses a deprecated method.
Is there any way to use the `@Deprecated` annotation in Processing?

Kind regards
macbrayne
